CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ if(WIN32)
 	enable_language(RC)
 	set(CMAKE_RC_COMPILER windres)
 	file(
-		DOWNLOAD "https://github.com/TheElixZammuto/sunshine-prebuilt/releases/download/1.0.0/pre-compiled.zip" "${CMAKE_CURRENT_BINARY_DIR}/pre-compiled.zip"
+		DOWNLOAD "https://github.com/caioavidal/sunshine-prebuilt/releases/download/1.0.0/pre-compiled.zip" "${CMAKE_CURRENT_BINARY_DIR}/pre-compiled.zip"
 		TIMEOUT 60
 		EXPECTED_HASH SHA256=5d59986bd7f619eaaf82b2dd56b5127b747c9cbe8db61e3b898ff6b485298ed6)
```

Comment:
Please adjust... See here:

https://github.com/SunshineStream/Sunshine/pull/50/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL35-R37

Reply:
I've added web ui tab for quicksync and also added to encoder's select. for now I need to keep this Download url because the new one from @PapyKahan doesn't have libmfx.

Reply:
Okay, that's fine for now